### ISSUE TITLE
Domain/protocol mismatch warning under phantomjs-1.9.8

### ISSUE THREAD

**Issue by eradman** [jeffschwartz/preamble#2]
Starting with phantomjs 1.9.8 a warning is issued when running tests based on
the test template:

    $ phantomjs-1.9.8 javascripts/phantom-runner.js index.html
    'waitFor()' finished in 2901ms.
    10 groups/16 tests/66 assertions passed
    Unsafe JavaScript attempt to access frame with URL about:blank from frame
    with URL file://javascripts/phantom-runner.js. Domains, protocols and ports
    must match.

While it still works this warning is very distracting when running tests! I
confirmed that 1.9.7 does not do this.

**Comment by jeffschwartz** [comment]
@eradman thanks. Elevated the issue to "bug".

**Comment by jeffschwartz** [comment]
@eradman what version of Preamble are you running?

**Comment by eradman** [comment]
Good question, I'm using preamble-v1.3.0-0-g1535e13

**Comment by jeffschwartz** [comment]
@eradman thanks.

**Comment by jeffschwartz** [comment]
@eradman as it isn't actually a bug since it worked with what was the latest version of Phantom at the time that Preamble v1.3 was released I've changed the status to "enhancement". Preamble v2 is ready to roll :) All that is missing is its documentation. Numerous enhancements in v2 like nested groups, an alternate and optional bdd syntax (the older tdd syntax is still fully supported), nicer/cleaner browser reports, full support for existing v1.3 test scripts. etc. As such Preamble v2.x will be targeted for the enhancement.
